semantic-source/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2015-2019 GitHub
```

Comment:
A lot of this code has been sitting around since 2015. (At a minimum, the `Range` stuff.)